server/services/index.ts

```diff
@@ -72,8 +73,11 @@ export const services = () => {
   const offencesPageService = new OffencesPageService(prisonApiClientBuilder)
   const offencesService = new OffencesService(prisonApiClientBuilder, calculateReleaseDatesApiClientBuilder)
   const personalPageService = new PersonalPageService(prisonApiClientBuilder, curiousApiClientBuilder)
+  const prisonService = new PrisonService(prisonRegisterStore, prisonRegisterApiClientBuilder)
+  const curiousService = new CuriousService(curiousApiClientBuilder, prisonService)
   const workAndSkillsPageService = new WorkAndSkillsPageService(
```

Comment:
At the moment the `WorkAndSkillsPageService` class takes both the client builder for the Curious REST API (to support it's existing calls to things like `/learnerProfile` etc, and the new `CuriousService`. As Curious API calls are migrated into service methods in `CuriousService` over time it should eventually mean that the client builder can be removed from `WorkAndSkillsPageService`